compiler/src/iree/compiler/Pipelines/Pipelines.h

```diff
@@ -60,6 +62,10 @@ inline static void enumerateIREEVMPipelinePhases(
            "Adjusts program ABI for the specified execution environment.");
   callback(IREEVMPipelinePhase::Preprocessing, "preprocessing",
            "Compiles up to the `preprocessing` specified");
+  callback(IREEVMPipelinePhase::FlowPreprocessing, "flow-preprocessing",
```

Comment:
I would rather not have a `"flow-preprocessing" phase, and just have a global optimization phase.